.gitignore

```diff
@@ -26,6 +26,7 @@ Obj/
 
 # Visual Studio 2015 cache/options directory
 .vs/
+.vscode/
```

Comment:
You had a previous commit where the files inside the `.vscode` were checked into the repository. So even though you've now added the `.vscode` folder to the `.gitignore`, it won't have an effect on the files already checked in. Google how to remove previously checked files that should have been ignored.